sdk/core/System.ClientModel/src/Options/ClientPipelineOptions.cs

```diff
@@ -23,6 +23,13 @@ public class ClientPipelineOptions
     private PipelineTransport? _transport;
     private TimeSpan? _timeout;
 
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+    public ClientPipelineOptions()
+#pragma warning restore CS1591 // Missing XML comment for publicly visible type or member
+    {
+        RetryPolicy = ClientRetryPolicy.Default;
```

Comment:
Setting the policy in the constructor means that a subtype would have to allocate a separate instance and override it in their base constructor.  Is that a problem?

Making the property on ClientPipelineOptions virtual instead has the negative of making the setter virtual too, which is hard to evolve.